test/features/steps/shared.js

```diff
@@ -29,6 +29,43 @@ var sharedSteps = function(){
   });
 
   /**
+   * Choose a CSV file to upload
+   *
+   * Usage: When I select a valid file to upload
+   * Usage: When I select an invalid file to upload
+   * @params {string} valid "a valid" if the file to be uploaded is valid
+   */
+  this.When(/^I select (a valid|an invalid) CSV file to upload$/, function(valid, next) {
+    var fileName = valid === 'a valid' ? 'valid.csv' : 'invalid.csv';
```

Comment:
This line's a bit… dense. Maybe add some parentheses to make it a little easier to parse?